tutorials/scripting/cross_language_scripting.rst

```diff
@@ -223,6 +235,9 @@ because no C# static types exist for signals defined by GDScript:
 
     myGDScriptNode.Connect("my_signal", Callable.From(MySignalHandler));
 
+    myGDScriptNode.Connect("my_signal_with_params", Callable.From(new System.Action<string, int>(MySignalWithParamsHandler)));
+    // Note that a System.Action must be explicitly created to avoid a "cannot convert from 'method group' to 'System.Action'" error.
```

Comment:
You can just specify the generic type arguments, hopefully a future .NET version can infer them.

```suggestion
    myGDScriptNode.Connect("my_signal_with_params", Callable.From<string, int>(MySignalWithParamsHandler));
```

Reply:
Amazing!